includes/emails/class-give-email-tags.php

```diff
@@ -1394,6 +1395,15 @@ function __give_211_bc_email_template_tag_param( &$args, $func_args = array() )
 			'is_admin' => false,
 		);
 	} else {
+
+		if ( array_key_exists( 'description', $args ) ) {
```

Comment:
@Sidsector9 please write comment on code that it is for backward compatibility use `desc` or `func` instead